releasenotes/notes/add-djm-enabled-config-667dd030d0bf7d83.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add new parameter `djm_config.enabled` to enable Data Jobs Monitoring
```

rst uses double backticks:

```suggestion
    Add new parameter ``djm_config.enabled`` to enable Data Jobs Monitoring
```